docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of Access Requests for Teleport roles. This
```

Comment:
> for Teleport roles.

Does this mean we don't support automatic reviews for resource access requests?

Edit: I now see this mentioned at the very end in the troubleshooting section. I feel like we should make this limitation more clear up front.

Reply:
I've updated the first section to include this notice:
```
Support for automatic reviews of resource Access Requests is currently in development.
```